collect_app/src/main/java/org/odk/collect/android/fragments/BarCodeScannerFragment.kt

```diff
@@ -105,24 +97,52 @@ abstract class BarCodeScannerFragment : Fragment() {
                 }
             }
 
-        barcodeScannerViewContainer.barcodeScannerView.start()
-        return rootView
+        binding.barcodeView.barcodeScannerView.start()
+        return binding.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        val binding = FragmentScanBinding.bind(view)
```

Comment:
Do you think creating binding in every method is a better approach than having one field, as shown here https://developer.android.com/topic/libraries/view-binding?